maps/tgstation.dmm

```diff
@@ -4029,7 +4029,7 @@
 "bzy" = (/obj/structure/disposalpipe/segment,/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden{dir = 4},/obj/machinery/atmospherics/pipe/simple/supply/hidden,/turf/simulated/floor{dir = 4; icon_state = "whiteblue"},/area/medical/medbay)
 "bzz" = (/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden{dir = 4},/obj/machinery/door/firedoor/border_only{dir = 8; name = "Firelock West"},/obj/machinery/door/airlock/glass_medical{id_tag = ""; name = "Paramedic Station"; req_access_txt = "500"},/turf/simulated/floor{icon_state = "white"},/area/medical/paramedics)
 "bzA" = (/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden{dir = 4},/turf/simulated/floor{icon_state = "white"},/area/medical/paramedics)
-"bzB" = (/obj/machinery/atmospherics/unary/vent_scrubber/on{dir = 8},/turf/simulated/floor{icon_state = "white"},/area/medical/paramedics)
+"bzB" = (/obj/machinery/atmospherics/unary/vent_scrubber/on{dir = 8},/mob/living/simple_animal/cat/snek/corpus,/turf/simulated/floor{icon_state = "white"},/area/medical/paramedics)
```

Comment:
They both hunt mice etc
They have literally no distinct functionality, so why not?
bear/skrite was ACTUALLY bad